### ISSUE TITLE
[Reports] Ability for the admin to save its rendering options per reports

### ISSUE THREAD

**Issue by RachL** [openfoodfoundation/wishlist#224]
## What is the need / problem?

As discussed here: https://github.com/openfoodfoundation/openfoodnetwork/pull/9032#issuecomment-1087592894 

Once this PR is merged, it would be nice to have the ability to save rendering options.

## Which type of users does this problem affect (and how many, if known)?

_- Hub Managers (MCFEs)_
_- Producers_
_- Instance Managers_

## Potential solutions that will solve the problem  ?[[brainstorming to list feature candidates]

A small checkbox? 

## Is Papercut or feature candidate?

_Add papercut or feature label to answer the question. If you are not sure, don't worry product team will cover :-)_

**Comment by RachL** [comment]
@jibees will check if this is a papercut or not. Proposed solution : always save, no checkbox. Saving is done when clicking on go.

**Comment by jibees** [comment]
I'd vote for a paper-cut in term of development. 
Creating all the specs for all the rendering options for each reports could be very time-consuming. Maybe we should assume to create only for the reports that had the greatest number of options?

**Comment by RachL** [comment]
@jibees I can be mistaken but it looks to me like all reports have the same number of options. Unless you count the ability to hide 1 column as one? In that case, it depends on the number of columns one report has.

**Comment by jibees** [comment]
You're right, it should consist to only create a spec for one report, and see if it's well saved.
Not so time consuming finally.
